zone/client.cpp

```diff
@@ -9514,63 +9514,202 @@ void Client::CheckVirtualZoneLines()
 
 void Client::ShowDevToolsMenu()
 {
-	std::string menu_commands_search;
-	std::string menu_commands_show;
-	std::string reload_commands_show;
-	std::string devtools_toggle;
+	std::string menu_search;
+	std::string menu_show;
+	std::string menu_reload_one;
```

Comment:
Not in love with the one, two etc.

We could vector string build and then build each display line dynamically based on number of elements

Fine for now

Reply:
That sounds good to me, only reason I did it this way was because I manually built the menu to where the stuff would fit well.